packages/cpp/src/browser/cpp-build-configurations-element.ts

```diff
@@ -0,0 +1,51 @@
+/********************************************************************************
+ * Copyright (C) 2018 Ericsson
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { injectable, inject } from 'inversify';
+import { StatusBar, StatusBarAlignment } from '@theia/core/lib/browser';
+import { CPP_CHANGE_BUILD_CONFIGURATION, CppBuildConfigurationManager } from './cpp-build-configurations';
+
+@injectable()
+export class CppBuildConfigurationsElement {
```

Comment:
I would suggest renaming this (and the file) to include "status bar", to make it clear it's a status bar element.  So `CppBuildConfigurationsStatusBarElement`, or something like that.